ddcz/views.py

```diff
@@ -346,17 +346,23 @@ def author_detail(request, author_id, slug):
 
 
 def phorum(request):
-    if request.method == "POST":
-        form = PhorumCommentForm(request.POST)
-        if form.is_valid() and request.user:
-            Phorum.objects.create(
-                reputace=0,
-                reg=1,
-                user=request.user.profile,
-                text=form.cleaned_data["text"],
-                nickname=request.user.profile.nick_uzivatele,
-                email=request.user.profile.email_uzivatele,
-            )
+    if request.method == "POST" and request.POST["post_type"] and request.user:
+        if request.POST["post_type"] == "d":
+            db_data = Phorum.objects.get(id=request.POST["post_id"])
```

Comment:
FWIW, with `Phorum.objects.delete(id=request.POST["post_id"], nickname = request.user.profile.nick_uzivatele)`, you would save yourself a query and a database roundtrip.

Reply:
I believe it needs to get the object first, otherwise it doesn't work (slack comment). I fixed all other notes, if you'll find this as done you can close it and merge, I think it is done otherwise. :)